### PR TITLE
add standalone-like class file

### LINKED ISSUE #14: Create LaTeXiT-sh class file
It would be useful to have a class file such that

* it takes a single block-boxes
* and creates a document with single page where page size fits the given block-boxes

We want to use such a class file when we create a figure with SATySFi which will be embedded in another document ofLaTeX or Word.

### PR DESCRIPTION
**Pull request by nyuichi**
closes #14

### CODE REVIEW

File: __test__/satysrc/satysfi-it.saty [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import: util
+
+satysfi-it ${1 + 2 = 3}
```
Comment (zeptometer) [inline __test__/satysrc/satysfi-it.saty]: NIT: Newline at EOF (so as other derive/02-assume.saty)

File: __test__/regression.test.js [hunk 2]
```diff
@@ -28,12 +28,18 @@ test('Confirm that satysfi is installed', () => {
...
 
 test('Check compiler outputs', () => {
-  const compilerOutput = shell.exec('satysfi satysrc/test.saty', {silent: true})
+  const compilerOutput = shell.exec('satysfi satysrc/generic.saty', {silent: true})
   .exec('awk \'/evaluating texts .../{flag=1;next}/evaluation done/{flag=0}flag\'', {silent: true})
   .stdout;
   expect(compilerOutput).toMatchSnapshot();
 })
 
+test('SATySFi-iT', async (done) => {
```
Comment (zeptometer) [inline __test__/regression.test.js]: 👍

File: __test__/satysrc/derive/01-derive.saty [hunk 3]
```diff
@@ -1,17 +1,11 @@
...
-  show-toc = false;
-|) '<
-  +math (${
-    \proven!(
-      open DeriveDSL in
-      derive ${A \vdash A}
-    )
-  });
->
+satysfi-it ${
```
Comment (zeptometer) [inline __test__/satysrc/derive/01-derive.saty]: Thank you for rewriting existing tests🙏

File: __test__/satysrc/satysfi-it.saty (hunk 1 above)
Comment (nyuichi) [reply __test__/satysrc/satysfi-it.saty]: done

**Pull request merged**